engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -881,8 +881,8 @@ protected boolean checkWorkItems(final VMInstanceVO vm, final State state) throw
             }
 
             if (state != State.Stopped) {
-                s_logger.debug("VM " + vm + " is not in a state to be started: " + state);
-                return null;
+                s_logger.warn("VM " + vm + " is not in a state to be started: " + state);
+                throw new CloudRuntimeException(String.format("Cannot start VM: %s in %s state", vm, state));
```

Comment:
We could unify these messages in a String var (with String.format).

Reply:
```suggestion
                String msg = String.format("Cannot start VM: %s in %s state", vm, state)
                s_logger.warn(msg);
                throw new CloudRuntimeException(msg));
```